ramls/users.raml

```diff
@@ -44,6 +44,25 @@ resourceTypes:
   post:
     is: [validate]
     description: Create a user
+  delete:
+    description: Delete a collection of users selected by a CQL query; this doesn't delete proxyFor records that reference them
+    is: [
+      language,
```

Comment:
Given that FOLIO is close to deciding to use a header for language localization, why introduce a query string parameter at this point that will likely need to be removed soon anyway?

Reply:
Removed. (This pull request is more than 3 months old.)

Reply:
Thank you for removing it